tests/unit_tests/test_game_rating.py

```diff
@@ -984,10 +1047,292 @@ async def test_single_wrong_report_still_rated_correctly(game: Game, player_fact
+                    2: pytest.approx((1889, 75), abs=1),
+                },
+                outcomes=IGNORED
+            ),
+            PersistedResults(
+                rating_type=RatingType.GLOBAL,
+                ratings={
+                    1: pytest.approx((2038, 348), abs=1),
+                    2: pytest.approx((1340, 419), abs=1),
+                },
+                outcomes=IGNORED
+            )
+        ]
+    )
+
+
+async def test_rating_adjustment_1v1_ladder_pros_global_mixed(
+    rating_service, player_factory
+):
+    IGNORED = object()
+
+    # Both players have high ladder rating, one has low global, the other has
+    # high global.
+    await do_test_rating_adjustment(
+        rating_service,
+        player_factory,
+        ratings={
+            1: {
+                RatingType.LADDER_1V1: Rating(2000, 75),
+                RatingType.GLOBAL: Rating(1000, 250),
+            },
+            2: {
+                RatingType.LADDER_1V1: Rating(1900, 75),
+                RatingType.GLOBAL: Rating(2100, 75),
+            }
+        },
+        expected_results=[
+            PersistedResults(
+                rating_type=RatingType.LADDER_1V1,
+                ratings={
+                    1: pytest.approx((2011, 75), abs=1),
+                    2: pytest.approx((1889, 75), abs=1),
+                },
+                outcomes=IGNORED
+            ),
+            PersistedResults(
+                rating_type=RatingType.GLOBAL,
+                ratings={
+                    1: pytest.approx((1373, 208), abs=1),
+                },
+                outcomes=IGNORED
+            )
+        ]
+    )
+
+
+async def test_rating_adjustment_1v1_ladder_pro_vs_global_pro(
+    rating_service, player_factory
+):
+    IGNORED = object()
+
+    # One player has high ladder rating the other has high global
+    await do_test_rating_adjustment(
+        rating_service,
+        player_factory,
+        ratings={
+            1: {RatingType.LADDER_1V1: Rating(2000, 75)},
+            2: {RatingType.GLOBAL: Rating(1900, 75)},
+        },
+        expected_results=[
+            PersistedResults(
+                rating_type=RatingType.LADDER_1V1,
+                ratings={
+                    1: pytest.approx((2003, 75), abs=1),
+                    2: pytest.approx((1340, 419), abs=1),
+                },
+                outcomes=IGNORED
+            ),
+            PersistedResults(
+                rating_type=RatingType.GLOBAL,
+                ratings={
+                    1: pytest.approx((1766, 429), abs=1),
+                },
+                outcomes=IGNORED
+            )
+        ]
+    )
+
+
+async def test_rating_adjustment_2v2_newbies(rating_service, player_factory):
+    IGNORED = object()
+
+    # Both players have no ratings yet
+    await do_test_rating_adjustment(
+        rating_service,
+        player_factory,
+        ratings={
+            1: {}, 3: {},
```

Comment:
Maybe it makes sense to give one of them a global rating to show how the system behaves